src/debugger/Debugger.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import type { BabelNode } from "babel-types";
+import { Breakpoint } from "./Breakpoint.js";
+import { ExecutionContext } from "../realm.js";
+import invariant from "../invariant.js";
+import fs from "fs";
+
+export class Debugger {
+  constructor(dbgFileLines: Array<string>) {
+    this.breakpoints = new Map();
+    this.prevBreakLine = 0;
+    this.prevBreakCol = 0;
+    this.parseCommands(dbgFileLines);
+  }
+  breakpoints: Map<number, Breakpoint>;
+  prevBreakLine: number;
+  prevBreakCol: number;
+
+  serializeContext(context: ExecutionContext) {
```

Comment:
What is the purpose of this method? It does not seem to be used anywhere? Also, mark it with leading underscore if private.

Reply:
This has been removed for now.